### PR TITLE
adds Dockerfile

### PR DESCRIPTION
**Pull request by andresgnlez**
Context: in case the application needs to be deployed in a non-Vercel platform, we must provide, at least, an alternative for it. I suggest going with a Dockerfile. 

This PR ensures the application can be run in a dockerized container. 

To test the PR:
1) ensure you have Docker installed
2) run `docker build -t {app_name} .` to build container and images
3) run `docker run -p 3000:3000 {app_name}`

The application should be available in `http://localhost:3000/` now.

Reference: the Dockerfile is a copy of the [one provided by NextJS](https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile).

This PR closes [this issue](https://github.com/Vizzuality/front-end-scaffold/issues/26).

### CODE REVIEW

**Comment by clementprdhomme** [comment]
Thanks for the mention! I have very limited knowledge of Docker so I'll let someone else review!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# Rebuild the source code only when needed
+FROM node:16-alpine AS builder
+WORKDIR /app
+COPY --from=deps /app/node_modules ./node_modules
+COPY . .
+
+# Next.js collects completely anonymous telemetry data about general usage.
+# Learn more here: https://nextjs.org/telemetry
+# Uncomment the following line in case you want to disable telemetry during the build.
+# ENV NEXT_TELEMETRY_DISABLED 1
```
Comment (davidsingal) [inline Dockerfile:22]: I prefer to disable the telemetry

Comment (davidsingal) [inline Dockerfile:22]:
```suggestion
ENV NEXT_TELEMETRY_DISABLED 1
```

I prefer to disable the telemetry

File: next.config.js [hunk 2]
```diff
@@ -10,6 +10,10 @@ const nextConfig = {
 
     return config;
   },
+  experimental: {
+    // https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files-experimental
+    outputStandalone: true,
```
Comment (davidsingal) [inline next.config.js:15]: is this required for Docker?? Did you try this new feature?? I prefer not to use experimental features in production

**Review by davidsingal — APPROVED** [review]

File: next.config.js (hunk 2 above)
Comment (andresgnlez) [reply next.config.js:15]: It's not. But it's the only way to run the application without additional setup.

The feature prepares the application in a single folder in a way it takes only the necessary to run independently, including a minimal server. That's why the last command of the Dockerfile (`CMD ["node", "server.js"]`) works.

If we remove it, the developer must develop a way to launch the application once compiled.

I have no issues removing the feature, but I think this is the only way to ensure the Dockerfile works and can be run without any additional configuration/files.

Comment (davidsingal) [reply next.config.js:15]: It's fine to me, let's leave it. In case something fails in the infrastructures, it will the first I'll check.

**Pull request merged**